chess_gui.py

```diff
@@ -178,86 +189,58 @@ def main():
         draw_game_state(screen, game_state, valid_moves, square_selected)
 
         endgame = game_state.checkmate_stalemate_checker()
-        if endgame == 0:
-            game_over = True
-            draw_text(screen, "Black wins.")
-        elif endgame == 1:
-            game_over = True
-            draw_text(screen, "White wins.")
-        elif endgame == 2:
-            game_over = True
-            draw_text(screen, "Stalemate.")
+        if endgame in range(3):
```

Comment:
What is this check for? You follow it up with nothing that wasn't accomplished before.